lib/diffit/timestamp.rb

```diff
@@ -0,0 +1,25 @@
+# encoding: utf-8
+
+module Diffit
+  # Provides a value converter to datetime
+  #
+  module Timestamp
```

Comment:
Separate module is a little bit too much for just a timestamp, maybe better to put it in some Utils module?